.github/workflows/marketplace-publish.yml

```diff
@@ -43,6 +47,7 @@ jobs:
           echo "$package" | grep -q "dist/extension.js" || exit 1
           echo "$package" | grep -q "extension/webview-ui/build/assets/index.js" || exit 1
           echo "$package" | grep -q "extension/node_modules/@vscode/codicons/dist/codicon.ttf" || exit 1
+          echo "$package" | grep -q ".env" || exit 1
```

Comment:
I'm assuming anyone can open up the .vsix and access this key. Does it have limited permissions? Just curious keys distributed to clients are support to work for Posthog.

Reply:
Posthog says:

> You can use this write-only key in any one of [our libraries](https://posthog.com/docs/libraries).
> Write-only means it can only create new events. It can't read events or any of your other data stored with PostHog, so it's safe to use in public apps.

Reply:
🙇  Makes sense!